app/src/main/java/com/example/parkingsystem/mvp/view/ParkingReservationView.kt

```diff
@@ -0,0 +1,62 @@
+package com.example.parkingsystem.mvp.view
+
+import android.app.Activity
+import android.app.DatePickerDialog
+import android.app.TimePickerDialog
+import android.icu.util.Calendar
+import com.example.parkingsystem.R
+import com.example.parkingsystem.databinding.ActivityParkingReservationBinding
+import com.example.parkingsystem.mvp.contract.ParkingReservationContract
+import com.example.parkingsystem.mvp.view.base.ActivityView
+import com.example.parkingsystem.util.showToast
+import java.util.Calendar.getInstance
+
+class ParkingReservationView(activity: Activity, private val binding: ActivityParkingReservationBinding) :
+    ParkingReservationContract.ParkingReservationView, ActivityView(activity) {
+
+    override fun getButtonPickerBegin(): Boolean = binding.editDateBegin.isPressed
+
+    override fun showDatePickerDialog(dateListener: DatePickerDialog.OnDateSetListener) {
+        val calendar = getInstance()
+        context?.let {
+            val datePicker = DatePickerDialog(
+                it,
+                dateListener,
+                calendar.get(Calendar.YEAR),
+                calendar.get(Calendar.MONTH),
+                calendar.get(Calendar.DAY_OF_MONTH)
+            )
+            datePicker.show()
```

Comment:
there's no need to create extra variables here, just use ...).show()

Reply:
Done